hera_pspec/uvwindow.py

```diff
@@ -1278,24 +1163,24 @@ def run_and_write(self, filepath, bl_lens, bl_weights=None,
             cyl_wf[ib, :, :, :] = self.get_cylindrical_wf(bl_lens[ib],
                                                           kperp_bins*self.kunits,
                                                           kpara_bins*self.kunits)*bl_weights[ib]
+        # cannot write None to file
+        if self.taper is None:
+            self.taper = 'none'
```

Comment:
I would not set this here, it would be surprising to the user. If you want to ensure the taper is always a string "none" instead of None, then you should validate that in the `__init__`. Otherwise when you *use* the taper you could just send in `self.taper or 'none'` instead of simply `self.taper`.

Reply:
Went for the second option!